shell/renderer/atom_renderer_client.cc

```diff
@@ -76,14 +76,26 @@ void AtomRendererClient::DidCreateScriptContext(
   // TODO(zcbenz): Do not create Node environment if node integration is not
   // enabled.
 
-  // Do not load node if we're aren't a main frame or a devtools extension
+  auto* command_line = base::CommandLine::ForCurrentProcess();
+
+   // Only load node if we are a main frame or a devtools extension
```

Comment:
urgh i think github's UI tricked me and inserted an extra space here.
```suggestion
  // Only load node if we are a main frame or a devtools extension
```